README.md

```diff
@@ -61,7 +61,9 @@ data from `localhost:8000` **where the Django app must be running**.
 ## CLI
 1. Run pipenv shell.
 2. Create a workflow using UI and save it. 
-3. Run it as: pyworkflow --file-directory (path-to-json-workflow-file) execute
+3. Run it as: pyworkflow execute (paths-to-json-workflow-files-separated-by-space)
```

Comment:
This can probably be simplified to `pyworkflow execute workflow-file ...`.

Separately, we should probably have a line or two to indicate reading/writing from stdin and stdout, or at least mention you can do so.